apps/web/playwright.config.ts

```diff
@@ -6,24 +6,32 @@ export default defineConfig({
   expect: {
     timeout: 5 * 1000,
   },
-  fullyParallel: true,
+  fullyParallel: false,
   forbidOnly: Boolean(process.env.CI),
   retries: process.env.CI ? 2 : 0,
   workers: process.env.CI ? 1 : undefined,
   reporter: [['html', { open: 'never' }]],
   use: {
-    baseURL: 'http://127.0.0.1:3000',
-    trace: 'on-first-retry',
+    baseURL: 'http://localhost:3000',
```

Comment:
What is the reason for `127.0.0.1` -> `localhost`? I am just curious, I don't have much experience with Playwright!

Reply:
these mean the same thing right @billc-dev ?

Reply:
They point to the same thing. I was just having trouble adding the cookie to `127.0.0.1`